.github/workflows/docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Docs
+on: [push, pull_request]
+env:
+  CI: true
+  NODE: 10.x
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set Node.js version
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ env.NODE }}"
+
+      - run: node --version
+      - run: npm --version
+      - run: java -version
+
+      - name: Set up npm cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-v${{ env.NODE }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}-${{ hashFiles('.github/workflows/docs.yml') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-v${{ env.NODE }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}-${{ hashFiles('.github/workflows/docs.yml') }}
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Test docs
+        run: npm run docs
```

Comment:
Shouldn't we generate dist files before running the docs task, to ensure the build is accurate?

Reply:
Not really since the docs tests don't depend on the dist files.